runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -271,4 +271,15 @@ default int insaneValuePrice()
 	{
 		return 10000000;
 	}
+
+	@ConfigItem(
+			keyName = "showRecolorItemPriority",
+			name = "Recolor Item Priority",
+			description = "Configures whether or not to recolor items over highlight list",
+			position = 21
+	)
```

Comment:
Step these back from two indents to one indent, so they match the other ConfigItems.